### ISSUE TITLE
[Bug] 3 in 1: Deleting Ship computer and replacing it seems to break Tutorial Mission; Deleting user does not reset tutorial progress

### ISSUE THREAD

**Issue by Michael-Phoenix** [tsunamayo/Starship-EVO#4275]
### Short Description of the issue:

I just found out, if you delete the first ship computer in the tutorial mission and place a new one, even when you connect the yoke, the mission seems to be broken. It does not realize that the yoke is connected. 
And even if you delete the user and make a new one, it seems to be stuck at the same "Connect the yoke to the computer" message

### Steps to reproduce the issue:

- make new player
- go through tutorial until you have to build the ship.
- build everything needed
- before connecting the yoke to the ship computer, delete the ship computer and place a new one
- tutorial is now stuck
- exit and delete player
- create new player with same name as the original
- you will now start the tutorial mission with the message "Connect the yoke to the ship computer"
- Deleting a user and making a new one with a different seed, also seems to remember the tutorial coordinates from previously existing user so the coordinates of tutorial waypoints do not align with the current seed.

### Environment
- Starship EVO version: 21w47b
- Graphics Card: nVIDIA 1070

**Comment by tsunamayo** [comment]
@Michael-Phoenix Hi, thanks a lot for booking the bug - indeed it makes sense. I will fix it asap!

**Comment by tsunamayo** [comment]
@Michael-Phoenix Issue is fixed, I have repushed the build so the fix is live now. Do you have anymore feedback with the tutorial (I am guessing you are a new player)
Especially do you think it was confusing to have to open the hatch and cockpit to build the system? Thanks!

**Comment by Michael-Phoenix** [comment]
@tsunamayo I am rather a returning player and I recognized the button visually from previous games.
The tutorial said something about the opening the rear hatch but something to visually mark the button would make things a lot easier for entirely new players.

Also the tutorial ended by me landing the ship after repairing the station modules. And I was quite confused as to the fact that there wasn't any "Congratulations you managed the tutorial, now go wild or go  do something else" message.
Maybe after landing give the player a confirmation that the tutorial really ends there. With the bugs I found earlier, I wasn't quite sure if I broke another thing or if it was indeed the end of the tutorial.

This one seems to have the same confusion:
https://github.com/tsunamayo/Starship-EVO/issues/4272#issuecomment-978201584_
